### PR TITLE
feature/FWF:3765-Added info component

### PR DESCRIPTION
**Pull request by fahad-aot**
# Issue Tracking

JIRA: 
Issue Type: BUG/ FEATURE

# Changes
1. Added custom info component and Updated screen with info component
2. Corrected decision table route 

# Screenshots (if applicable)
image
image

# Checklist
- [ ] Updated changelog
- [ ] Added meaningful title for pull request

### CODE REVIEW

**Review by abilpraju-aot — APPROVED** [review]

File: forms-flow-web/src/components/Modeler/DecisionTable.js [hunk 1]
```diff
@@ -32,7 +32,12 @@ const DecisionTable = React.memo(() => {
   const [searchDmnLoading, setSearchDmnLoading] = useState(false);
   const redirectUrl = MULTITENANCY_ENABLED ? `/tenant/${tenantKey}/` : "/";
   const [search, setSearch] = useState(searchText || "");
-
+  
+  useEffect(() => {
+    if (!search?.trim()) {
+      dispatch(setBpmnSearchText(""));
```
Comment (arun-s-aot) [inline forms-flow-web/src/components/Modeler/DecisionTable.js:38]: @fahad-aot can u check this lint error. seems like this varible is not declared

File: forms-flow-web/src/components/Modals/ImportModal.js [hunk 2]
```diff
@@ -114,19 +114,10 @@ const ImportModal = React.memo(({ importFormModal, onClose, formSubmitted,
...
                         {importError && importError.includes("already exists") &&
-                            <div className="import-error-note">
-                                <div className="d-flex gap-2 align-items-center">
-                                    <IButton />
-                                    <Translation>{(t) => t("Note")}</Translation>
-                                </div>
-                                <div>
-                                    <Translation>
-                                        {(t) => t(`If you want to replace an existing form,
+                        <CustomInfo Heading="Note" Content='If you want to replace an existing form,
```
Comment (arun-s-aot) [inline forms-flow-web/src/components/Modals/ImportModal.js:117]: pls cross check this . casing of 'heading' and 'content' is wrong

Comment (fahad-aot) [reply forms-flow-web/src/components/Modals/ImportModal.js]: Updated.

File: forms-flow-web/src/components/Modeler/DecisionTable.js (hunk 1 above)
Comment (fahad-aot) [reply forms-flow-web/src/components/Modeler/DecisionTable.js]: Fixed

**Pull request merged**